### PR TITLE
EPMFARMATS-12674

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12674.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Calculator {
+    constructor() { };
+
+    add() {
```
Comment (SiarheiRylach) [inline EPMFARMATS-12674.js:4]: Sugar: can be used spread operator ```...params```

[the remark this answers is not in the document]
Comment (overdd) [reply EPMFARMATS-12674.js]: Rewrited using ...params

File: EPMFARMATS-12674.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        let result = null;
+        for (let i = 0; i < arguments.length; i++) {
+            result += +arguments[i];
+        }
+        return result;
+    };
+
+    multiply() {
```
[the remark this answers is not in the document]
Comment (overdd) [reply EPMFARMATS-12674.js]: Rewrited using ...params

File: EPMFARMATS-12674.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        if (arguments.length === 0) {
+            let result = null;
+            return result;
+        } else {
+            let result = 1;
+            for (let i = 0; i < arguments.length; i++) {
```
[the remark this answers is not in the document]
Comment (overdd) [reply EPMFARMATS-12674.js]: Rewrited

File: test/specs/add.spec.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    let calculator;
+    beforeEach(() => {
+        calculator = new Calculator();
+    });
+
+    it(`should return null if result if arguments are empty`, () => {
+        expect(calculator.add()).to.be.equal(null);
+    });
+
+    it(`should return NaN if result if arguments are empty are strings`, () => {
```
[the remark this answers is not in the document]
Comment (overdd) [reply test/specs/add.spec.js]: Removed

File: EPMFARMATS-12674.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+class Calculator {
+    constructor() { };
+
+    add() {
+        let result = null;
+        for (let i = 0; i < arguments.length; i++) {
```
[the remark this answers is not in the document]
Comment (overdd) [reply EPMFARMATS-12674.js]: Rewrited

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**